packages/block-editor/src/components/rich-text/index.js

```diff
@@ -89,6 +88,20 @@ const globalStyle = document.createElement( 'style' );
 
 document.head.appendChild( globalStyle );
 
+function createPrepareEditableTree( props ) {
+	const fns = Object.keys( props ).reduce( ( accumulator, key ) => {
+		if ( key.startsWith( 'format_prepare_functions' ) ) {
+			accumulator.push( props[ key ] );
+		}
+
+		return accumulator;
+	}, [] );
+
+	return ( value ) => fns.reduce( ( accumlator, fn ) => {
```

Comment:
Small typo: I believe it should be `accumulator`.

Reply:
Fixed in 382baa8. Thanks!